server/graphql/resolvers.js

```diff
@@ -91,14 +91,14 @@ module.exports = {
       dataSources.ssbFlumeAPI.getAllSettlements(),
     activeProposals: (_, __, { dataSources }) =>
       dataSources.ssbFlumeAPI.getActiveReciprocityProposals(),
-    profile: async (_, { id }) => {},
+    profile: (_, { id }) => getProfile(id),
     decryptMagicLink: async (_, { hash }) => {
       const [public, private, email] = decrypt(hash).split('~~');
       return {
         public,
         private,
         curve: 'ed25519',
-        id: public,
+        id: `@${public}`,
```

Comment:
just curious if you can explain what is going on here to me @luandro ?

Reply:
The decryption returns the public key, they id is basically the public key with a @ in front. Having the proper id makes calls much easier.